sdk/src/main/scala/com/horizen/websocket/server/WebSocketServerEndpoint.scala

```diff
@@ -192,10 +208,14 @@ private object WebSocketServerEndpoint extends ScorexLogging {
   }
 
   def notifyMempoolChanged(): Unit = {
-    val eventPayload = sidechainNodeChannelImpl.getRawMempool().get
-    this.sessions.forEach(session =>{
-        WebSocketServerEndpoint.sendMessage(EVENT_MESSAGE.code, -1, 2, eventPayload, session)
-    })
+    val eventPayload = sidechainNodeChannelImpl.getRawMempool() match {
+      case Success(eventPayload) =>
+        this.sessions.forEach(session =>{
+          WebSocketServerEndpoint.sendMessage(EVENT_MESSAGE.code, -1, 2, eventPayload, session)
+        })
+      case Failure(ex)  => log.error("Error on notifyMempoolChanged!: "+ex.toString)
+    }
+
```

Comment:
In the method below:  `def notifySemanticallySuccessfulModifier(): Unit` instead of `getBestBlock().get` use `match` as you did above.